src/core/scopes_configs.cc

```diff
@@ -623,6 +623,13 @@ int32_t scopes_configs::generate_alert(waflz_pb::alert** ao_alert,
         // -------------------------------------------------
         l_at->mutable_req_info()->set_customer_id(a_cust_id);
         // -------------------------------------------------
+        // set account_type
+        // -------------------------------------------------
+        cust_id_scopes_map_t::iterator i_scopes;
+        i_scopes = m_cust_id_scopes_map.find(a_cust_id);
```

Comment:
need a sanity check here. if i_scopes != m_cust_id_scopes_map.end(), then the logic to set account type. also could you rename the variable l_account_type to l_limit.